File: 백준/Bronze/10809. 알파벳 찾기/알파벳 찾기.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
	public static void main(String[] args) throws IOException {
		
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		StringBuilder sb = new StringBuilder();

		String str = br.readLine();

		for(int i=0; i < 26; i++){
			char ch = (char)(i + 'a');
			int n = str.contains(ch+"") ? str.indexOf(ch) : -1;
			sb.append(n).append(" ");
		}

		System.out.println(sb);

		br.close();
	}
}

File: 백준/Bronze/10809. 알파벳 찾기/알파벳 찾기.kt
import java.io.BufferedReader
import java.io.InputStreamReader

fun main(){
    val input = BufferedReader(InputStreamReader(System.`in`))
    val S = input.readLine()
    val alphabets = Array(26){-1}
    var index = 0
    for(c in S.toCharArray()){
        if(alphabets[c.code-97] == -1) alphabets[c.code-97] = index
        index++
    }
    alphabets.forEach{
        v -> print("$v ")
    }
}